heat/core/tests/test_suites/basic_test.py

```diff
@@ -0,0 +1,173 @@
+from unittest import TestCase
+from heat.core import dndarray, MPICommunication, MPI, types, factories
+import heat as ht
+
+import numpy as np
+import torch
+
+
+class BasicTest(TestCase):
+
+    __comm = MPICommunication()
```

Comment:
This does not seem like a good idea. The comm might be dependent on the input variable

Reply:
So the comm should rather be passed via the assert functions? For the array_equal function I understand the point but for the func_equal, the input values don't have a comm.